.github/workflows/auto-tidy.yml

```
@@ -0,0 +1,23 @@
+name: auto-tidy
+on:
+  schedule:
+    - cron: '* * * * 1' # Mondays https://youtu.be/2WJYA9YDWi4
+
+jobs:
+  auto-tidy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        ref: ${{ github.head_ref }}
+    - uses: actions/setup-go@v2
+      with:
+        go-version: '^1.14.0'
+    - uses: evantorrie/mott-the-tidier@v1-beta
+      with:
+        gomods: |
+          **/go.mod
+          -tools/go.mod
+    - uses: stefanzweifel/git-auto-commit-action@v4
+      with:
+        commit_message: '(auto-commit) go mod tidy'
```

I'd be more comfortable if we did some kind of sanity test first, even if it's as simple as `go install` to prove it compiles, before committing back.

---

Honestly I was just gonna let it fail (apparently... every minute on Monday. Yikes) and fix it later.